src/utilities.ts

```diff
@@ -171,12 +171,14 @@ function getWordAtPosition(document, position) {
     return text;
 }
 
-function getDocument(document): vscode.TextDocument {
+function getDocument(
+    document: vscode.TextDocument | Record<string, never>
+): vscode.TextDocument {
     if (
         document &&
         Object.prototype.hasOwnProperty.call(document, 'fileName')
     ) {
-        return document;
+        return document as vscode.TextDocument;
```

Comment:
Typescript can't figure out that this a `TextDocument` without a type asserting function or adding an "as".